cloud.ganttproject.colloboque/src/main/kotlin/cloud/ganttproject/colloboque/DevServer.kt

```diff
@@ -43,7 +54,8 @@ class DevServerMain : CliktCommand() {
     val updateInputChannel = Channel<InputXlog>()
     val dataSourceFactory = PostgresDataSourceFactory(pgHost, pgPort, pgSuperUser, pgSuperAuth)
     val colloboqueServer = ColloboqueServer(dataSourceFactory::createDataSource, initInputChannel, updateInputChannel)
-    ColloboqueHttpServer(port, colloboqueServer).start(NanoHTTPD.SOCKET_READ_TIMEOUT, false)
+    ColloboqueHttpServer(port, colloboqueServer).start(0, false)
```

Comment:
it doesn't matter for the dev purposes, but setting a timeout for the regular HTTP connection makes sense, because it is not supposed to be open for long time.